dts/arm64/intel/intel_socfpga_agilex5.dtsi

```diff
@@ -97,6 +97,10 @@
 		reg = <0x80100000 DT_SIZE_M(8)>;
 	};
 
+	fpga0: bridges {
```

Comment:
Shouldn't there be an address associated with this device inside of the fpga SoC?

Reply:
Same for here